### PR TITLE
fix: goimports failed in make all

### PR DESCRIPTION
**Pull request by rayowang**
**What this PR does**:
I also encountered a similar problem before. After I cloned the registry from github, it was ok to map it to the registry corresponding to golang.org. But a better solution might be to use a good VPN.
**Which issue(s) this PR fixes**:
#569

### CODE REVIEW

File: make/golang.mk [hunk 1]
```diff
@@ -120,12 +120,35 @@ go.style:
...
+	@GO111MODULE=off $(GO) build $(GOPATH)/src/golang.org/x/tools/cmd/goimports
+	@GO111MODULE=off $(GO) install $(GOPATH)/src/golang.org/x/tools/cmd/goimports
 endif
 
 .PHONY: go.format
 go.format: go.format.verify
 	@echo "===========> Running go codes format"
 	$(GO_FMT) -s -w .
-	$(GO_IMPORTS) -w -local $(GO_MODULE) .
+	$($GOPATH/bin/GO_IMPORTS) -w -local $(GO_MODULE) .
```
Comment (Xunzhuo) [inline make/golang.mk:153]: This is incorrect maybe, try:
```
	$(GOPATH)/bin/$(GO_IMPORTS) -w -local $(GO_MODULE) .
```

**Review by Xunzhuo — COMMENTED** [review]
Thanks @rayowang ,but actually this is not what I expect.

In the new commands, we will clone three repos which are not in small size too and it contains some tools we do not need, and we need to link them(Isn`t it too complex?). 

Why do not we just use go get goimports to just install goimports? We will also face the network issue again even if we choose to clone it from github. Dealing with the network issue should not be a task from the team in my view.

File: make/golang.mk (hunk 1 above)
Comment (rayowang) [reply make/golang.mk]: I'm sorry, I didn't pay attention to this, thank you for correcting the mistake.

**Comment by rayowang** [comment]
> seeflood



> Thanks @rayowang ,but actually this is not what I expect.
> 
> In the new commands, we will clone three repos which are not in small size too and it contains some tools we do not need, and we need to link them(Isn`t it too complex?).
> 
> Why do not we just use go get goimports to just install goimports? We will also face the network issue again even if we choose to clone it from github. Dealing with the network issue should not be a task from the team in my view.

**Comment by rayowang** [comment]
> Thanks @rayowang ,but actually this is not what I expect.
> 
> In the new commands, we will clone three repos which are not in small size too and it contains some tools we do not need, and we need to link them(Isn`t it too complex?).
> 
> Why do not we just use go get goimports to just install goimports? We will also face the network issue again even if we choose to clone it from github. Dealing with the network issue should not be a task from the team in my view.

I just saw the information, I think what you said makes sense, but as seeflood @seeflood  said, it is impossible to access golang.org in China. In fact, some VPN tools may not work. The person who asked the question before has been doing it for a long time. Of course, it is better to build in docker, but this will lose the convenience of local build, so this method is a compromise at present.

**Comment by seeflood** [comment]
@Xunzhuo Any idea on this PR?
I'm ok with it.
An alternative is we first ping golang.org . If the ping returns successfully, we can download the goimports from golang.org directly.
Anyway, both solutions are ok for me.

**Review by Xunzhuo — APPROVED** [review]

**Comment by Xunzhuo** [comment]
I am Okay with this, thanks @rayowang

**Review by seeflood — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**